apps/multiplatform/common/src/commonMain/resources/MR/base/strings.xml

```diff
@@ -2220,4 +2220,13 @@
   <string name="download_errors">Download errors</string>
   <string name="server_address">Server address</string>
   <string name="open_server_settings_button">Open server settings</string>
+
+  <!-- ContactsView.kt -->
+  <string name="contact_type_new">New</string>
+  <string name="contact_type_deleted">Deleted contacts</string>
```

Comment:
contact_type_deleted, contact_type_recent and icon_descr_pending_contact_connection seem to be unused translations - better to delete them so that translator don't have to translate them into all languages